Includes/Runeterra/Commons/Constants.hpp

```diff
@@ -13,6 +13,9 @@ constexpr int NUM_ALL_CARDS = 844;
 //! The max known version.
 const int MAX_KNOWN_VERSION = 2;
 
+//! The number of players.
+const int NUM_PLAYERS = 2;
```

Comment:
Are there reasons for "MAX_KNOWN_VERSION" and "NUM_PLAYERS" are not "constexpr" variables?

Reply:
Oh, this is my mistake. I'll fix it! 🧷